res/database/Default/database_programmes.xml

```diff
@@ -1809,18 +1901,20 @@ Anrührender Film, geplant als TV-Serie - es blieb beim Piloten.</de>
 			<data country="USA" year="1954" distribution="1" maingenre="17" subgenre="" flags="0" blocks="2" price_mod="1" />
 			<ratings critics="75" speed="51" outcome="84" />
 		</programme>
-		<programme id="a1d45a76-b16f-43b0-aa9d-7fe8778be885" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0065718" creator="">
+		<programme id="a1d45a76-b16f-43b0-aa9d-7fe8778be885" product="1" licence_type="1" tmdb_id="0" imdb_id="tt0065718" creator="" comment="International title 'The Fire-Tongue Bowl' seems to be more common for the 1970 version, but keeping the 1944 title for continuity">
 			<title>
-				<de>Die Streichholzhammerbowle</de>
-				<en></en>
+				<de>Die Streichholzhammerbowle (1970)</de>
```

Comment:
I think, the year was left out on purpose. So you would have to look at the attributes...
leave the difference more subtle?

Reply:
please check, if leaving out the year for Streichholzhammerbowle would be acceptable